### DISCUSSION TITLE
Making Chakra CLI not add "forwardRef" anymore with React 19

### DISCUSSION THREAD

**Discussion by lcswillems** [chakra-ui/chakra-ui#9325] [Ideas]
### Description

React 19 is now out and forwardRef is not necessary anymore as "ref" can be passed as a normal prop.

However, Chakra CLI continues to add forwardRef in the code it generates.

It would be great if the Chakra CLI was not adding forwardRef in the code it generates with Chakra 19.

### Problem Statement/Justification

Chakra CLI continues to add forwardRef in the code it generates even though React 19 doesn't require it anymore.

### Proposed Solution or API

It would be great if the Chakra CLI was not adding forwardRef in the code it generates with Chakra 19.

**Comment by segunadebayo** [comment]
Snippets are copies. Edit them. `forwardRef` is still valid on React 19.
